corfudb-tools/src/main/java/org/corfudb/browser/CorfuBrowserMain.java

```diff
@@ -0,0 +1,26 @@
+package org.corfudb.browser;
+
+import java.util.Map;
+
+import org.corfudb.util.GitRepositoryState;
+import org.docopt.Docopt;
+
+public class CorfuBrowserMain {
+    private static final String USAGE = "Usage: corfu-browser --host=<host> " +
+        "--port=<port> --namespace=<namespace> --tablename=<tablename>\n"
+        + "Options:\n"
+        + "--host=<host>   Hostname\n"
+        + "--port=<port>   Port\n";
```

Comment:
Add options for namespace and table name as well

Reply:
done